pom.xml

```diff
@@ -174,10 +173,8 @@
         <version.org.jboss.security.jboss-negotiation>2.3.7.Final</version.org.jboss.security.jboss-negotiation>
         <version.org.jboss.security.jbossxacml>2.0.8.Final</version.org.jboss.security.jbossxacml>
         <version.org.jboss.shrinkwrap.shrinkwrap>1.1.2</version.org.jboss.shrinkwrap.shrinkwrap>
-        <version.org.jboss.shrinkwrap.descriptors>2.0.0-alpha-6</version.org.jboss.shrinkwrap.descriptors>
```

Comment:
temporary replaced with 2.0.0-alpha-7 until we align versions in wildfly-arquillian.